modules/globebrowsing/src/timequantizer.cpp

```diff
@@ -215,36 +209,35 @@ void DateTime::incrementOnce(int value, char unit) {
     bool inBounds = true;
     switch (unit) {
         case 'm':
-            if (singleIncrement(_minute, value, 0, 59))
+            if (singleIncrement(_minute, value, 0, 59)) {
                 break;
-            // else fall-through if overflow...
-
+            }
+            //[[ fallthrough ]]
```

Comment:
Why add the comment to these lines? Without it compiler warnings about fallthrough should avoided.

Reply:
Oopsie